modules/vpc/main.tf

```diff
@@ -0,0 +1,27 @@
+module "vpc" {
+  source  = "terraform-aws-modules/vpc/aws"
+  version = "~> 2.63"
+
+  name = var.name
+  cidr = "10.${var.subnet}.0.0/16"
+
+  azs                 = var.azs
+  private_subnets     = ["10.${var.subnet}.1.0/24", "10.${var.subnet}.2.0/24", "10.${var.subnet}.3.0/24"]
+  public_subnets      = ["10.${var.subnet}.101.0/24", "10.${var.subnet}.102.0/24", "10.${var.subnet}.103.0/24"]
```

Comment:
In the interest of matching our other environments, I changed these third octets but this might not be something we'll be able to push to the nerves-hub-terraform master repo. I suppose we could 1). use the ones from master, 2). make these variables as well.

Reply:
Making them variables could work, or at that point maybe make the subnet cidr lists variables since that will be the same number of variables.

Reply:
I made the lists variables and also cidr a variable 👍